[2024-08-04T13:17:58.756+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-04T13:17:58.788+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: export_multiple_dbs_to_s3_dag.extract_and_upload_users scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T13:17:58.797+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: export_multiple_dbs_to_s3_dag.extract_and_upload_users scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T13:17:58.798+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2024-08-04T13:17:58.814+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_and_upload_users> on 2024-08-03 00:00:00+00:00
[2024-08-04T13:17:58.822+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=80) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-04T13:17:58.823+0000] {standard_task_runner.py:64} INFO - Started process 82 to run task
[2024-08-04T13:17:58.824+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'export_multiple_dbs_to_s3_dag', 'extract_and_upload_users', 'scheduled__2024-08-03T00:00:00+00:00', '--job-id', '272', '--raw', '--subdir', 'DAGS_FOLDER/export_multiple_dbs_to_s3_dag .py', '--cfg-path', '/tmp/tmp_1_2bjv0']
[2024-08-04T13:17:58.825+0000] {standard_task_runner.py:91} INFO - Job 272: Subtask extract_and_upload_users
[2024-08-04T13:17:58.867+0000] {task_command.py:426} INFO - Running <TaskInstance: export_multiple_dbs_to_s3_dag.extract_and_upload_users scheduled__2024-08-03T00:00:00+00:00 [running]> on host 8ccbadcdc1e3
[2024-08-04T13:17:58.945+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='export_multiple_dbs_to_s3_dag' AIRFLOW_CTX_TASK_ID='extract_and_upload_users' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-03T00:00:00+00:00'
[2024-08-04T13:17:58.946+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-04T13:17:59.550+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-04T13:17:59.550+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-04T13:17:59.559+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=export_multiple_dbs_to_s3_dag, task_id=extract_and_upload_users, run_id=scheduled__2024-08-03T00:00:00+00:00, execution_date=20240803T000000, start_date=20240804T131758, end_date=20240804T131759
[2024-08-04T13:17:59.599+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-04T13:17:59.613+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-04T13:17:59.615+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
